### DISCUSSION TITLE
Cannot get a new installation going

### DISCUSSION THREAD

**Discussion by Dennis3321** [mampfes/hacs_waste_collection_schedule#2875] [Q&A]
Im on the latest HAOS, install the HACS component, try to add the integration, fill in the details, using GAD in the Netherlands. Cannot submit get a "unknown error". 

I leave everything as default as possible. Cannot trouble shoot. I don't see any requirements. Is it maybe because my postcode / house number is not included? Does it rely on other integrations (calander?)?

![Screenshot_20241019-085806](https://github.com/user-attachments/assets/02a5bdbf-d2ee-4de1-b647-93d0696612e0)

**Comment by 5ila5** [comment]
The source you're using (hvcgroep_nl) set The TITLE parameter weirdly, so the config_flow (GUI Configuration fails), I fixed this in #2879


This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
